gtfs_realtime_translators/factories/factories.py

```diff
@@ -12,23 +12,31 @@ def create(entity_id, **kwargs):
 class TripUpdate:
 
     @staticmethod
-    def __get_stop_time_events(arrival_time, departure_time=None):
+    def __get_stop_time_events(arrival_time, departure_time, arrival_delay, departure_delay):
         arrival = gtfs_realtime.TripUpdate.StopTimeEvent(time=arrival_time)
         if departure_time is None:
             departure = arrival
         else:
             departure = gtfs_realtime.TripUpdate.StopTimeEvent(time=departure_time)
-
+        if arrival_delay is not None:
+            arrival = gtfs_realtime.TripUpdate.StopTimeEvent(delay=arrival_delay)
+            if departure_delay is None:
+                departure = arrival
+            else:
+                departure = gtfs_realtime.TripUpdate.StopTimeEvent(delay=arrival_delay)
```

Comment:
Repetitive looking code. Can this be moved to a function ?

Reply:
Agree. I think we could have a helper here that sets a single StopTimeEvent